cli/detc.py

```diff
@@ -8,6 +8,27 @@ def cli():
   """Welcome to the DETC deployment tool"""
   pass
 
+@cli.command()
+@click.argument("cloud")
+@click.argument("action")
+def traffic(cloud, action):
+  """Lacework: Deploy Agents Pods
+
+    \b
+    ACTION
```

Comment:
Would be good to add a plan action here similar to the `eks terraform` command so you can take a look before launching